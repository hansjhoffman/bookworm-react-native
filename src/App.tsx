import React from "react";
import { ApolloClient } from "apollo-client";
import { InMemoryCache } from "apollo-cache-inmemory";
import { HttpLink } from "apollo-link-http";
import { ApolloProvider } from "@apollo/react-hooks";

import { BootstrapLoader } from "./navigation/bootstrap_loader";

const client = new ApolloClient({
  cache: new InMemoryCache(),
  link: new HttpLink({ uri: "https://bookworm.hansjhoffman.dev/graphql" }),
});

export default function App() {
  return (
    <ApolloProvider client={client}>
      <BootstrapLoader />
    </ApolloProvider>
  );
}
