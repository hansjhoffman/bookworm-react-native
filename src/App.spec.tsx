import React from "react";
import { render } from "@testing-library/react-native";

import App from "./App";

describe("App", function() {
  it("finds text", function() {
    const { getByText } = render(<App />);

    getByText("Hello World");
  });

  it("matches snapshot", function() {
    const { baseElement } = render(<App />);

    expect(baseElement).toMatchSnapshot();
  });
});
