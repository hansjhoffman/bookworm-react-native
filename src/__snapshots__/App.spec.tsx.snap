// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text>
      Hello World
    </Text>
  </View>
</View>
`;
